src/containers/login/LoginPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import LoginForm from './../login/LoginForm';
+import './login.css';
+
+const LoginPage = () => (
+    <div className="login-page">
+        <div className="content-form">
```

Comment:
It think it will be neat and more responsive if you use Rows and Cols to display the information on this page.